src/Module.Server/Common/Models/CrpgAgentApplyDamageModel.cs

```diff
@@ -202,6 +202,45 @@ public override float GetDamageMultiplierForBodyPart(BoneBodyPartType bodyPart,
         defenderStunMultiplier = 1f;
     }
 
+    // Consider reworking once https://forums.taleworlds.com/index.php?threads/missioncombatmechanicshelper-getdefendcollisionresults-bypass-strikemagnitudecalculationmodel.459379 is fixed
```

Comment:
maybe declare this as a summary
could be easier to find it again.
Alternatively add a "TODO: " as prefix to search for todos in the future